### DISCUSSION TITLE
How to config hive connector to use aliyun oss cloud storage?

### DISCUSSION THREAD

**Discussion by ChenShuai1981** [trinodb/trino#13992] [Q&A]
I'm going to use trino to do some operations (create/insert/update/delete) on delta lake which store data files on aliyun oss cloud storage[1]. But I can not find related document about how to config it except for aws s3, azure storage and gcs. Could you do me a favor? Thanks a lot!

1. https://www.aliyun.com/product/oss/

**Comment by Ardentlyer** [comment]
Hello, have you solved the problem of hive configuring OSS?
